lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -11724,6 +11724,32 @@ void Emit(ParseNode* pnode, ByteCodeGenerator* byteCodeGenerator, FuncInfo* func
         ENDSTATEMENET_IFTOPLEVEL(isTopLevel, pnode);
         break;
     }
+    // The Coalescing operator resolves to the left hand side if it is not null or undefined
+    // In that case the right hand side is not evaluated
+    // If the left hand side is null or undefined it resolves to the right hand side
+    // PTNODE(knopCoalesce     , "??"        ,None    ,Bin  ,fnopBin)
+    case knopCoalesce:
```

Comment:
This case in the bytecode emitter is the run time implementation - the command is fully de-sugared here, so no new opcodes were needed.